### PR TITLE
Initial doc updates

### PR DESCRIPTION
**Pull request by roblg**
Updating some documentation.

### CODE REVIEW

**Comment by gigabo** [comment]
So many `triton`s to deal with...

File: packages/react-server/README.md [hunk 1]
```diff
@@ -82,24 +82,24 @@ Read more on [our dev blog](https://www.redfin.com/devblog/2015/09/thoughts-on-t
 	And add the minimal server logic:
 
 		// "use strict" required to use `class` and `let`, but not strictly required
-		// for using Triton.
+		// for using react-server.
 		"use strict";
 
 		// Require dependencies
-		let triton = require("triton"),
+		let rs = require("react-server"),
```
Comment (gigabo) [inline packages/react-server/README.md]: `rs`, huh?  I like it.

Comment (roblg) [inline packages/react-server/README.md]: Seemed simple enough

File: packages/react-server/docs/why-triton.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+When we talk about performance in `react-server`, we're mostly talking about _perceived performance_. We believe that [WebPageTest](http://www.webpagetest.org/) is correct to focus on how much time it takes from the moment the user navigates the browser until the content above the fold is painted. There are other important perceived performance metrics (like time to first content painted, time to all content painted, and time to full interactivity), but in developing react-server, we have focused most on reducing time to above-the-fold paint. WebPageTest's [Speed Index](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/metrics/speed-index) is probably the best current measure of above-the-fold paint speed, as it takes into account how quickly the content is painted during load, not just the total time to full completion.
```
Comment (gigabo) [inline packages/react-server/docs/why-triton.md]: These lines are too long to get within-line change highlighting.  Would be nice to break them up at some point.

**Pull request merged**